nestor_api/lib/k8s/builders.py

```diff
@@ -1,12 +1,39 @@
 """k8s deployment file builders"""
 
+import os
 from typing import Optional
 
+from pybars import Compiler  # type: ignore
+
 from nestor_api.config.probes import ProbesDefaultConfiguration
 from nestor_api.config.replicas import ReplicasDefaultConfiguration
 import nestor_api.lib.docker as docker
+import nestor_api.lib.io as io
 import yaml_lib
 
+TEMPLATES = [
+    "deployment",
+    "hpa",
+    "anti-affinity-node",
+    "anti-affinity-zone",
+    "service",
+    "namespace",
+    "cronjob",
+    "job",
+]
```

Comment:
```suggestion
TEMPLATES = [
    "deployment",
    "hpa",
    "anti-affinity-node",
    "anti-affinity-zone",
    "service",
    "namespace",
    "cronjob",
]
```

Reply:
I will actually need the `job` to build the cronjob.